BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -152,9 +152,18 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     if !isLocal {
       publisherView.publisherNameLabel.text = state.dataSource?.displayString(for: state.url)
       
-      // FIXME: Remove fake data
-      publisherView.setVerified(true)
-      attentionView.valueLabel.text = "19%"
+      guard let host = state.url.host else { return }
+      
+      state.ledger.publisherInfo(forId: host) { info in
+        guard let publisher = info else { return }
+        
+        DispatchQueue.main.async {
```

Comment:
We should probably jump to main from the actual ledger method call site instead of at UI level (should technically always execute completion blocks on main unless manually specified)